ui_framework/src/global_styling/reset/_reset.scss

```diff
@@ -25,6 +25,7 @@ time, mark, audio, video {
   border: 0;
   font-size: 100%;
   font: inherit;
+  font-family: $kuiFontFamily;
```

Comment:
This overrides the default font-family Chrome applies to the button element via the font property.